_posts/2021-01-16-sr2021-first-league-sessions.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: news
+title: SR2021 First League Session a Great Success!
+---
+
+After the first 36 matches of the SR2021 Competition, team HRS3 from
+Hills Road Sixth Form College head up the league leaderboard with a
+staggering 82 game points. They're closely followed by the other 3 Hills Road
+teams as well as one of our rookie teams We Robots.
```

Comment:
It would be great to get in somewhere early that this is merely the first of many league sessions. Currently it would be easy to read this part and assume that the league stage was now over.

Reply:
https://github.com/srobo/website/pull/253/commits/452ab977fb83033950a04194bf08237c45364a70